genai-on-vertex-ai/vertex-ai-search/Ingesting_unstructured_docuemtns_with_metadata.ipynb

```diff
@@ -0,0 +1,1054 @@
+{
```

Comment:
<ol><li>Clarify schema is for the metadata and not the actual document itself.</li><li>re: <code>gives special retrieval treatment</code> - can we add/link to this from the documentation</li></ol>

<pre class="ql-syntax" spellcheck="false"> Note: Key properties are can improve the quality of search and recommendations results and search autocomplete accuracy. If you use auto-schema detection, key properties are not automatically added.
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/applied-ai-engineering-samples/pull/67/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>Clarified #1.</p><p>Unfortunately we don't have any documentation on keyproperties AFAIK</p>